.gitignore

```diff
@@ -1,3 +1,6 @@
+# project specific
+example/*
```

Comment:
Are you sure we don't want to leave `main_prompt`?
maybe we should use a `.keep` file or specify in the `gitignore` that we want to ignore everything EXCEPT for the `main_prompt` file

Reply:
Leaving the main_prompt out would help with reducing conflicts when people are testing or just using the repo. Every time you go to do a git pull to get the latest update you'll end up with merge conflicts... At least that's my 2¢

Reply:
Fair enough, I tend to agree.

Users will only use the example once to try, so yeah, no biggie.